lib/get/get-full-source-space.js

```diff
@@ -166,5 +166,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => item.sys.publishedVersion)
+  return includeDrafts ? items : items.filter((item) => !!item.sys.publishedVersion)
```

Comment:
not really an issue but this just making sure that we return `true` or `false` explicitly